crates/router/src/routes/dummy_connector/consts.rs

```diff
@@ -2,6 +2,7 @@ pub const PAYMENT_ID_PREFIX: &str = "dummy_pay";
 pub const ATTEMPT_ID_PREFIX: &str = "dummy_attempt";
 pub const REFUND_ID_PREFIX: &str = "dummy_ref";
 pub const DEFAULT_RETURN_URL: &str = "https://app.hyperswitch.io/";
+pub const ASSETS_BASE_URL: &str = "https://app.hyperswitch.io/assets/TestProcessor/";
```

Comment:
Can we move this in the configuration of dummy connector (environment variables)